src/containerapp/azext_containerapp/commands.py

```diff
@@ -120,6 +120,11 @@ def load_command_table(self, _):
         g.custom_command('set', 'set_ingress_traffic', exception_handler=ex_handler_factory())
         g.custom_show_command('show', 'show_ingress_traffic')
 
+    with self.command_group('containerapp ingress ip-restriction') as g:
+        g.custom_command('set', 'set_ip_restriction', exception_handler=ex_handler_factory())
+        g.custom_command('remove', 'remove_ip_restriction')
```

Comment:
Do we need to consider adding `confirmation=True` for the remove command?

Reply:
This is a property, and keeping with the convention in containerapp commands I think it's unnecessary

Reply:
OK